CHANGELOG.md

```diff
@@ -5,4 +5,4 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [3.0.1](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/compare/3.0.0...3.0.1) - 2016-05-22
 ### Fixed
 - Added missing default sort by primary key(s).
-- Allow non integer/string types as identifier (ex. uuid). 
+- Allow non integer/string types as identifier (ex. uuid).
```

Comment:
This is introduced by this PR commit: https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/567/commits/1001034f4b2a41ff40c65c4a730df67508f97740#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR8

Just squash them and get rid of this.